ECIPs/ecip-1056.md

```diff
@@ -10,18 +10,25 @@ created: 2019-03-25
 
 ### Simple Summary
 
-Enable the outstanding Ethereum Foundation _Constaninople_ and _Petersburg_ network protocol upgrades on the Ethereum
-Classic network in a hard-fork code-named _Agharta_ to enable maximum compatibility across these networks.
+Enable the outstanding Ethereum Foundation _Constaninople_ and _Petersburg_ EVM opcode upgrades, while also introducing a _Generalized Account Versioning Scheme_ for the EVM. 
+
+These changes would be made on the Ethereum Classic network in a hard-fork code-named _Agharta_ to enable maximum compatibility across EVM-based networks and to ensure backwards-compatiblity for existing contracts across hardforks modifying VM rules.
 
 ### Abstract
 
 Add support for a subset of protocol-impacting changes introduced in the Ethereum Foundation (ETH) network via the
-_Constaninople_ and _Petersburg_ hardforks. The proposed changes for Ethereum Classic's _Agharta_ upgrade include:
+_Constaninople_ and _Petersburg_ hardforks. The proposed changes for Ethereum Classic's _Agharta_ upgrade include the following opcode introductions:
 
 - Constantinople bitwise shifting instructions
 - Constantinople skinny `CREATE2` opcode
 - Constantinople `EXTCODEHASH` opcode
 
+Further, add support for:
+ - Generalized Account Versioning [EIP-1702: Generalized Account Versioning Scheme](https://eips.ethereum.org/EIPS/eip-1702), which "defines a method of hard forking while maintaining the exact functionality of existing account by allowing multiple versions of the virtual machines to execute in the same block."
+ 
+ 
+#### Schedule
+
 This document proposes the following blocks at which to implement these changes in the Classic networks:
 
 - `1_500_000` on Kotti Classic PoA-testnet (early November 2019)
```

Comment:
can you also update that?

Potential Agharta timeline

- Morden Testnet (_deprecated_), Dec 11, 2019, 12pm UTC, block `5_348_435` (Gethc, Parity)
- Mordor Testnet, Jan 15, 2020, 12pm UTC, block `617_716` (Gethc, Parity, Mgeth)
- Kotti Testnet, Feb 05, 2020, 12pm UTC, block `2_028_202` (Parity, Mgeth)
- Classic Mainnet, Apr 15, 2020, 12pm UTC, _no block number required yet_

Reply:
yep, thanks, done, and will close #36